dashboard/app/views/levels/editors/fields/_long_instructions.html.haml

```diff
@@ -13,7 +13,26 @@
     %div{ style: 'background-color: rgb(239, 239, 239); padding-top: 12px' }
       ~ f.text_area :long_instructions, rows: 4
 
+- if @level.is_a? GamelabJr
+  = f.label :instructions_icon,'Instructions Icon'
+  %p Please refer to #{link_to 'Sprite Lab Static Assets', 'https://github.com/code-dot-org/code-dot-org/tree/staging/apps/static/spritelab'} for available assets, then enter the name of the image to use as the instructions icon for this level.
+  = f.text_field :instructions_icon, value: @level.instructions_icon
+  %p Preview:
+  %img.instructions-icon-preview{style: 'height: 64px;'}
+
 :javascript
+  $(document).ready(function() {
+    function updateIconPreview() {
+      var iconName = $('#level_instructions_icon')[0].value;
+      if (iconName) {
+        $('.instructions-icon-preview').attr('src', 'https://studio.code.org/blockly/media/spritelab/' + iconName + '.png');
+      } else {
+        $('.instructions-icon-preview').attr('src', 'https://studio.code.org/blockly/media/spritelab/avatar.png');
+      }
```

Comment:
nit: i think this would be more readable if we set a fallback for iconName, which is essentially what we're doing below, but would allow us to set the src attribute once

```javascript
var iconName = $('#level_instructions_icon')[0].value || 'avatar';
$('.instructions-icon-preview').attr('src', 'https://studio.code.org/blockly/media/spritelab/' + iconName + '.png');
```